templates/bundle/resources/usr/local/bin/monitor_liferay_lifecycle.sh

```diff
@@ -11,7 +11,17 @@ function generate_thread_dump {
 
 	echo -e "${thread_dump}" > "${file_name}"
 
-	echo '"{'$(cat "${file_name}")'"}' > /proc/1/fd/1
+	# https://docs.cloud.google.com/logging/quotas#:~:text=Size%20of%20a,256%C2%A0KB1
+	# >  Size of a LogEntry:	256 KB (1)
+	# >  (1) This approximate limit is based on internal data sizes, not the actual REST API request size.
+	#
+	# Let's keep the limit smaller (100KB) for other log fields to fit and prefix with line number for sorting
```

Comment:
@topolik, is it really necessary?

BChan prefers to not have comments in the code unless it's really demanded.